analyzers/tests/SonarAnalyzer.UnitTest/TestCases/AppSettings/DatabasePasswordsShouldBeSecure/UnexpectedContent/ListKind/appsettings.json

```diff
@@ -0,0 +1,15 @@
+﻿{
```

Comment:
The folder is named "ListKind", but we have all kinds of types here. Maybe "PropertyKinds" is a bit better as a folder name.